at-hoangtran/Week4/Final/Migrate/migrate.rb

```diff
@@ -0,0 +1,60 @@
+require 'mysql2'
+require_relative '../Migrate/connect_db'
+class Migrate
+  attr_accessor :client
+  def initialize; end
+
+  def self.create_database
+    $client.query('CREATE DATABASE TranHoang_HumanResources;')
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/8bc115c94428b11d6e3406f6ef06575baf3e6ffc/at-hoangtran/Week4/Final/Migrate/migrate.rb#L8"></span>Style/GlobalVars: Do not introduce global variables. (https://github.com/rubocop-hq/ruby-style-guide#instance-vars, http://www.zenspider.com/Languages/Ruby/QuickRef.html)</td>
    </tr>
  </tbody>
</table>